.circleci/config.yml

```diff
@@ -17,7 +17,9 @@ aliases:
       paths:
         - ../.npm
 
-  e2e_steps: &e2e_steps
+commands:
```

Comment:
@znarf here is the `commands` stanza that I mention in my comment https://github.com/opencollective/opencollective-frontend/pull/3062#discussion_r352174632